.evergreen/utils.sh

```diff
@@ -26,12 +26,11 @@ createvirtualenv () {
     # Upgrade to the latest versions of pip setuptools wheel so that
     # pip can always download the latest cryptography+cffi wheels.
     PYTHON_VERSION=$(python -c 'import sys;print("%s.%s" % sys.version_info[:2])')
-    if [[ $PYTHON_VERSION == "3.4" ]]; then
-        # pip 19.2 dropped support for Python 3.4.
-        python -m pip install --upgrade 'pip<19.2'
-    elif [[ $PYTHON_VERSION == "2.7" || $PYTHON_VERSION == "3.5" ]]; then
-        # pip 21 will drop support for Python 2.7 and 3.5.
-        python -m pip install --upgrade 'pip<21'
+    if [[ $PYTHON_VERSION == "2.7" || $PYTHON_VERSION == "3.4" || $PYTHON_VERSION == "3.5" ]]; then
+        # Use get-pip for EOL Python versions.
+        curl -sSL https://bootstrap.pypa.io/pip/${PYTHON_VERSION}/get-pip.py -o get-pip.py
+        python get-pip.py
+        rm get-pip.py
```

Comment:
@behackett do you see any issue with the above approach? This was the only way I could seem to resolve the pip errors for Python 3.4 on macos.

Reply:
You can simplify this a bit, but the idea is sound. This is what we do when building the toolchain:
`curl --retry 3 -L https://bootstrap.pypa.io/pip/$PYTHON_VERSION/get-pip.py | $PYTHON_BINARY`

Reply:
Simplified.